src/main/java/net/snowflake/ingest/connection/TelemetryService.java

```diff
@@ -86,6 +96,14 @@ public void reportLatencyInSec(
     }
   }
 
+  // TODO @rcheng question - is it useful to report all the blob metadata here, or just latencies?
```

Comment:
Is blob metadata already reported elsewhere? If so do we need to report this telemetry? I'm worried about overloading the telemetry service with too many messages. Alternatively we could buffer or aggregate messages, but then the message size might be too large.